packages/teams-js/src/public/authentication.ts

```diff
@@ -668,6 +668,34 @@ export namespace authentication {
      * Limited to Microsoft-internal use
      */
     ver: string;
+    /**
+     * @hidden
+     * Stores the data residency of the user.
+     *
+     * @internal
+     * Limited to Microsoft-internal use
+     */
+    dataResidency?: DataResidency;
+  }
+
+  /**
+   * Limited set of data residencies information exposed to 1P application developers
+   */
+  export enum DataResidency {
```

Comment:
Since data Residency is marked internal and hidden up in L678, does it make sense to mark the interface hidden/internal as well?